pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[project]
+name = "gamatrix"
+version = "1.4.3"
+authors = [
+    {name = "Erik Niklas", email = "github@bobanddoug.com"},
+    {name = "Derek Keeler", email = "34773432+derek-keeler@users.noreply.github.com"},
+    {name = "Klosteinmann", email = "34807323+klosteinmann@users.noreply.github.com"},
+]
+description = """A tool to compare the games owned by several users, and list
+                 all the games they have in common. It requires all users to
+                 use [GOG Galaxy](https://www.gog.com/galaxy); since GOG Galaxy
+                 supports almost all major digital distribution platforms."""
+readme = "README.md"
+requires-python = ">=3.9"
+license = { file = "LICENSE" }
+
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
```

Comment:
I don't really understand this, but I assume 3.7 and 3.8 should be added.

Reply:
[Classifiers in PyPI](https://pypi.org/classifiers/) - I did steal this part from another project that does release to PyPI and even though we don't (and maybe never will), I thought I'd leave it. I'm not going to lose any sleep over it if you want it gone, total [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).

For now, I've updated it to include 3.7 & 3.8.